pavelib/utils/test/suites/nose_suite.py

```diff
@@ -128,7 +128,7 @@ def __init__(self, *args, **kwargs):
         if self.processes is None:
             # Use one process per core for LMS tests, and no multiprocessing
             # otherwise.
-            self.processes = 0
+            self.processes = -1 if self.root == 'lms' else 0
```

Comment:
@efischer19 try adding it as a command line option to your paver command, either as -p or --processes
e.g. `paver test_system -s lms --fasttest -p 0